source/server/util.js

```diff
@@ -1,4 +1,6 @@
 import { SUITES } from './types.js';
+import { WORLD_EVENTS } from './types.js';
+import { MULTIPLER } from './types.js';
```

Comment:
`import { SUITES, WORLD_EVENTS, MULTIPLIER } from './types.js';`

instead of three separate lines

Reply:
and multiplier is misspelled